plugins/catalog-graph/EXPERIMENTAL.md

```diff
@@ -0,0 +1,224 @@
+| catalogGraph  | Regular route  | Reference to the Catalog graph page route  |
+| catalogEntity | External route | Reference to the Catalog entity page route |
+
+With this, you can optionally associate the entity relations graph page to a different route other than the Catalog plugin's default entity page (for more information about route binding, see [this](https://backstage.io/docs/frontend-system/architecture/routes#binding-external-route-references) documentation).
+
+```yaml
+# example binding the catalog graph entity page
+app:
+  routes:
+    bindings:
+      # defaults to catalog.catalogEntity
+      catalog-graph.catalogEntity: <some-plugin-id>.<some-external-route-key>
+```
+
+You can also point an external route from another plug-in to the Catalog chart page:
+
+```yaml
+app:
+  routes:
+    bindings:
+      <some-plugin-id>.<some-external-route-key>: catalog-graph.catalogEntity
+```
+
+## Extensions
+
+This plugin extensions are configurable through the `app-config.yaml` file and their implementation are also overridable.
+
+### Catalog Entity Relations Graph Card
+
+A [Entity Card](https://backstage.io/docs/frontend-system/building-plugins/extension-types#entitycard---reference) extension renders the relation graph for a Catalog entity on the Catalog entity page. This extension has an action that redirects users to the Catalog entity's relations graph [page](#catalog-entity-relations-graph-page).
+
+| kind        | namespace     | name             | id                                           |
+| ----------- | ------------- | ---------------- | -------------------------------------------- |
+| entity-card | catalog-graph | entity-relations | `entity-card:catalog-graph/entity-relations` |
+
+#### Output
+
+A React component defined by the [coreExtensionData.reactElement](https://backstage.io/docs/reference/frontend-plugin-api.coreextensiondata/) type.
+
+#### Inputs
+
+There are no inputs available for this extension.
+
+#### Config
+
+The card configurations should be defined under the `app.extensions.entity-card:catalog-graph/entity-relations.config` key:
+
+```yaml
+# example configuring the card title
+app:
+  extensions:
+    # this is the extension id and it follows the naming pattern bellow:
+    # <extension-kind>/<plugin-namespace>:<extension-name>
+    - entity-card:catalog-graph/entity-relations:
+        config:
+          # defaults to "Relations"
+          title: 'Entities Relations Graph'
+```
+
+See below the complete list of available configs:
+
+| Key             | Description                                                                                                                                        | Type                                                                                                        | Optional | Default value                                                                                                               |
```

Comment:
I would appreciate someone more experienced with the Catalog plugin to help me write better descriptions for these configs.